apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -127,8 +127,10 @@ suite('navigation >', function() {
     suite('init() >', function() {
       test('display the right panel', function(done) {
         assert.isFalse(elements.InboxView.classList.contains('panel-active'));
+        assert.equal(elements.InboxView.getAttribute('aria-hidden'), 'true');
         Navigation.init().then(() => {
           assert.ok(elements.InboxView.classList.contains('panel-active'));
```

Comment:
It shouldn't be a string, it should be boolean :) http://mxr.mozilla.org/mozilla-central/source/dom/webidl/DOMTokenList.webidl#17 :)